code/index.html

```diff
@@ -4,27 +4,87 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Oxygen:wght@300&display=swap" rel="stylesheet">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+    <!-- Image-header and image-text -->
+    <header class="image-header"> 
+      <div class="header-text">
+        <h1> Time for adventure!</h1>
+        <p> Hike the mountains and the woods. <br> 
+        Explore our nature with us and learn more about it.<br>
+        It´s good for you!</p>
+      </div>
+    </header>
 
+    <!-- Sections with the rest of the content -->
+    <section class="content">
     <!-- Signup form -->
+      <section class="signup-form">
+        <div class="headers">
+          <h2>Wanna join?</h2>
+          <h3>Sign up for more info!</h3>
+        </div> 
+        <form id="form" action="https://httpbin.org/anything" method="post"> 
+          <div class="inputBoxes">
+            <div class="text-input">
+              <label>Name
+              <input type="name" name="name" id="name" required>
+              </label>
+              <label>E-mail
+              <input type="email" name="email" id="email" required>
+              </label>
+            </div>
+            <div class="checkboxes">
+              <label class="checkbox-container">
+              I agree to the Terms & Conditions
```

Comment:
Shouldn't this be in maybe a <p>? I looks like you can style it anyway, so just wondering how it works.